docs/en/integrations/elastic-package.asciidoc

```diff
@@ -0,0 +1,247 @@
+[[elastic-package]]
+= `elastic-package` reference
+
+`elastic-package` is a command line tool, written in Go, used for developing Elastic packages.
+It can help you lint, format, test, build, and promote your packages.
+
+// Currently, elastic-package only supports packages of type Elastic Integrations.
+
+[discrete]
+[[elastic-package-start]]
+== Get started
+
+. Download and build the latest master of elastic-package binary:
++
+[source,terminal]
+----
+git clone https://github.com/elastic/elastic-package.git
+make build
+----
++
+TIP: Make sure that you've correctly set up the https://golang.org/doc/gopath_code.html#GOPATH[`$GOPATH` and `$PATH`]
+environment variables. `elastic-package` must be accessible from your `$PATH`.
+
+. Change into the directory of the package under development:
++
+[source,terminal]
+----
+cd my-package
+----
+
+. Run the `help` command to see available commands
++
+[source,terminal]
+----
+elastic-package help
+----
+
+[discrete]
+[[elastic-package-command-reference]]
+== Command reference
+
+The following `elastic-package` commands are available.
+For more details on a specific command, run `elastic-package help <command>`.
+
+Some commands have a _global context_, meaning that they can be executed from anywhere.
+Other commands have a _package context_; these must be executed from somewhere under a package's
+root folder, and the command will only operate on the contents of that package.
+
+// *************************
+// The following is copied directly from
```

Comment:
Hi @bmorelli25 ,

I see that you copied content from elastic-package READMEs. Do you have a plan for updating it in the future? We used to modify README files from time to time, so these `.asciidoc`s will become outdated really soon.

Reply:
Hi @mtojek! Sorry for the delay. A long PTO break and other projects have gotten in the way. Returning to this now.

> Do you have a plan for updating it in the future?

Concrete plan, no. Eventually, we'd like the only documentation for these commands to exist in the Integrations developer guide. We discussed documenting these commands in a yaml file in the elastic-package repository with a script that can be used to extract and build to mdx. But that's still a pipe dream at this point.